charts/lagoon-logging/templates/logs-dispatcher.fluent-conf.configmap.yaml

```diff
@@ -299,6 +299,8 @@ data:
         tls_verify_hostname {{ . }}
         {{- end }}
         # endpoint
+        keepalive true # makes sure the connection is not recreated every second
+        keepalive_timeout 10m # reconnect after 10mins in order to handle DNS changes, etc.
```

Comment:
The problem with this is that it breaks load-balancing on the target forwarder because a standard kubernetes load balancer does round-robin on the targeted endpoints. So e.g. with this setting logs will flow to a single pod, switching pods every 10m.

Reply:
As discussed, maybe we could make this 1m to avoid loading a single pod for too long?

Reply:
Will leave it, can always tweak should issues arise.